packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -475,19 +532,117 @@ class _ModalBottomSheetRoute<T> extends PopupRoute<T> {
+
+  /// The z-coordinate at which to place this material relative to its parent.
+  ///
+  /// This controls the size of the shadow below the material.
+  ///
+  /// Defaults to 0, must not be negative.
   final double? elevation;
+
+  /// The shape of the bottom sheet.
+  ///
+  /// Defines the bottom sheet's [Material.shape].
+  ///
+  /// If this property is not provided, it falls back to [Material]'s default.
   final ShapeBorder? shape;
+
+  /// {@macro flutter.material.Material.clipBehavior}
+  ///
+  /// Defines the bottom sheet's [Material.clipBehavior].
+  ///
+  /// Use this property to enable clipping of content when the bottom sheet has
+  /// a custom [shape] and the content can extend past this shape. For example,
+  /// a bottom sheet with rounded corners and an edge-to-edge [Image] at the
+  /// top.
+  ///
+  /// If this property is null, the [BottomSheetThemeData.clipBehavior] of
+  /// [ThemeData.bottomSheetTheme] is used. If that's null, the behavior defaults to [Clip.none]
+  /// will be [Clip.none].
   final Clip? clipBehavior;
+
+  /// Defines minimum and maximum sizes for a [BottomSheet].
+  ///
+  /// Typically a bottom sheet will cover the entire width of its
+  /// parent. Consider limiting the width by setting smaller constraints
+  /// for large screens.
+  ///
+  /// If null, the ambient [ThemeData.bottomSheetTheme]'s
+  /// [BottomSheetThemeData.constraints] will be used. If that
+  /// is null, the bottom sheet's size will be constrained
+  /// by its parent (usually a [Scaffold]).
+  ///
+  /// If constraints are specified (either in this property or in the
+  /// theme), the bottom sheet will be aligned to the bottom-center of
+  /// the available space. Otherwise, no alignment is applied.
   final BoxConstraints? constraints;
+
+  /// Specifies the color of the modal barrier that darkens everything below the
+  /// bottom sheet.
+  ///
+  /// Defaults to `Colors.black54` if not provided.
   final Color? modalBarrierColor;
+
+  /// Specifies whether the bottom sheet will be dismissed
+  /// when user taps on the scrim.
+  ///
+  /// If true, the bottom sheet will be dismissed when user taps on the scrim.
+  ///
+  /// Defaults to true.
   final bool isDismissible;
+
+  /// Specifies whether the bottom sheet can be dragged up and down
+  /// and dismissed by swiping downwards.
+  ///
+  /// If true, the bottom sheet can be dragged up and down and dismissed by
+  /// swiping downwards.
+  ///
+  /// Defaults is true.
   final bool enableDrag;
+
+  /// The animation controller that controls the bottom sheet's entrance and
+  /// exit animations.
+  ///
+  /// The BottomSheet widget will manipulate the position of this animation, it
+  /// is not just a passive observer.
   final AnimationController? transitionAnimationController;
+
+  /// {@macro flutter.widgets.DisplayFeatureSubScreen.anchorPoint}
   final Offset? anchorPoint;
+
+  /// If useSafeArea is true, a SafeArea is inserted.
```

Comment:
```suggestion
  /// If useSafeArea is true, a [SafeArea] is inserted.
```

... to make dartdoc link to it